Sprint-2/implement/bmi.js

```diff
@@ -13,3 +13,11 @@
 // Given someone's weight in kg and height in metres
 // Then when we call this function with the weight and height
 // It should return their Body Mass Index to 1 decimal place
+
+function  bmi_Calculation(height, weight) {
+    let squareHeight = height * height;
+    
+    return  weight / squareHeight
+}
+
+console.log(bmi_Calculation(1.73, 70));
```

Comment:
When choosing a name for a function, it's best to use a verb / action word (see above "capitalise", "convert..."). So here you could call it "calculateBmi" instead. 
We also tend to use kebabCase, which doesn't use underscores

Reply:
done